### PR TITLE
Add in configuration table for unattended install

### PR DESCRIPTION
**Pull request by warrenbuckley**
Hi @sofietoft one for you & the docs team to consider adding

### CODE REVIEW

File: Getting-Started/Setup/Install/Unattended-Install.md [hunk 1]
```diff
@@ -73,3 +73,13 @@ After completing the 3 steps above you can now initialize the installation by bo
 Once it has completed, you should see the following when visiting the frontend of the site.
 
 ![Frontend of Umbraco site installed using the unattended installs feature](images/unattended/final-screen.png)
+
+## Configuration options
+
+|   	|  Configured Version 	|  With Conn String |  InstallUnattended 	|
```
Comment (sofietoft) [inline Getting-Started/Setup/Install/Unattended-Install.md:79]:
```suggestion
|   	|  Set ConfigurationStatus 	|  Include connectionstring |  InstallUnattended value 	|  
```

**Review by sofietoft — COMMENTED** [review]
Thanks for the suggstions @warrenbuckley !

I think it's a great idea with the table showing the configuration options.
However, I was a bit confused about the naming you gave the columns, so I've tried making it a bit more "true to the configs" - what they're actually called in the web.config file.

Let me know what you think!

**Comment by warrenbuckley** [comment]
Looks good to me 👍

**Review by sofietoft — APPROVED** [review]
Brilliant! Thanks for applying the suggestions.

I'll merge it right in 💪

**Pull request merged**